docs/book/src/introduction.md

```diff
@@ -11,9 +11,17 @@ Started by the Kubernetes Special Interest Group (SIG) [Cluster Lifecycle](https
 * [Developer guide](./developer/guide.md)
 * [Contributing](./CONTRIBUTING.md)
 
-**Using Cluster API v1alpha3 or v1alpha4?** Please see the corresponding documentation:
-* [release-0-3.cluster-api.sigs.k8s.io](https://release-0-3.cluster-api.sigs.k8s.io)
+<aside class="note">
+
+<h1>Documentation of older ClusterAPI versions</h1>
+
+This book documents ClusterAPI v1.2. For older Cluster API versions please see the corresponding documentation:
+* [release-1-1.cluster-api.sigs.k8s.io](https://release-1-1.cluster-api.sigs.k8s.io)
```

Comment:
Not sure if we should add a link to the `main` book as well, to make it very obvious.

Reply:
I'm +1 to add it, so when this will be server at https://cluster-api.sigs.k8s.io/, user know how to jump to the latest and greatest

Reply:
Done, yup I expect that I will need that myself as I'm still used to that whatever we just merged is already available on  https://cluster-api.sigs.k8s.io/